frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -221,7 +221,7 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
                                                 </div>
                                             )}
                                             {survey.type !== SurveyType.API ? (
-                                                <div className="mt-6">
+                                                <div className="mt-6" style={{ maxWidth: 320 }}>
```

Comment:
there should be a `max-w` class in tailscale

Reply:
True, but those are relative lengths. Here, I just copied what we do in SurveyEdit.tsx:915 - the form has a width of 320px, so the dropdown should span that too.